src/test/resources/org/jenkinsci/plugins/casc/BackwardCompatibilityTest.yml

```diff
@@ -0,0 +1,8 @@
+jenkins:
```

Comment:
I guess the parser is a bit too permissive, but this does not seem like valid yaml as it's missing the initial `---`

Reply:
This is not required by the yaml spec, only optional (mostly for a single file can contain multiple documents)  :http://yaml.org/spec/1.1/current.html section 2.2